docs/Configuration.md

```diff
@@ -712,6 +713,36 @@ For example, if you want to respect Browserify's [`"browser"` field](https://git
 }
 ```
 
+By combining `defaultResolver` and `packageFilter` we can implement a `package.json` "pre-processor" that allows us to change how the default resolver will resolve modules. For example, imagine we want to use the field `"module"` if it is present, otherwise fallback to `"main"`:
```

Comment:
As a sanity check, you've verified these instructions work in a real project as well? It seems reasonable, tho 🙂

Reply:
I did. Literally copy-pasted the example from the module resolver I'm writing, works as expected.

Reply:
wonderful, thanks!